### DISCUSSION TITLE
How to backup/restore recent & favorite repos?

### DISCUSSION THREAD

**Discussion by klorinczi** [gitextensions/gitextensions#8470] [General] [answered]
How to backup/restore recent & favorite repos?
Where is it stored? In Registry?
How can I move them to a portable Git Extensions?

**Comment by pmiossec** [comment, marked as answer]
I can't verify because I am away from a computer but I think it is stored
in `%AppData%\Local\Gitextensions\Gitextensions.settings`

**Reply by klorinczi** [reply]
Thanks for the info!

Finally I found Gitextensions.settings in directory where GitExtensions was installed. 
Likely it was a portable install.

So I copied Gitextensions.settings to another portable Git Extensions dir.

**Comment by mstv** [comment]
The recent repos are stored in the folder `%AppData%\GitExtensions\GitExtensions\Recent\`. Just copy the folder or the `.gitext` files you like.

**Reply by RussKie** [reply]
These are Windows MRU (i.e. _most recently used_), I think the Q was about the recent repositories in the dashboard.
